src/scripts/components/Wallet.js

```diff
@@ -10,9 +9,9 @@ import { ACTIVATE_SECURE_WALLET } from 'constants/ParatiiLibConstants'
 type Props = {
   user: Object,
   userAddress: string,
-  isWalletSecured: boolean,
+  isWalletSecured: string,
```

Comment:
Hm why is this a `string`? If we are passing in a `string` I think _that_ is the problem. We shouldn't change this flow type

Reply:
this is from @eliawk 
In fact wasn't to be here. Let me change